Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -698,8 +695,7 @@ namespace OpenInfraPlatform {
 						const EXPRESSReference<typename IfcEntityTypesT::IfcLoop>& loop = bound->Bound;
 						bool polyOrientation = bound->Orientation;
 						if (!loop) {
-							BLUE_LOG(warning) << "FaceConverter Problem with Face #" << faceID << ": IfcLoop #" << loop->getId() << " no valid loop.";
-
+							throw oip::InconsistentGeometryException(face, " no valid loop.");

```

Comment:
code immediatly following a `throw` is **never** executed. (a `throw` is similar to `return` in this case. What is it with the `if`-clause?

Reply:
See the newer comment which includes code improvement.

Reply:
Resolved:  d142297